Sources/IDEUtils/SyntaxClassifier.swift

```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+@_spi(RawSyntax) @_spi(SyntaxData) import SwiftSyntax
```

Comment:
I think `SyntaxData.contextualClassification` doesn't need to be `SyntaxData`, could you rewrite it to `Syntax.contextualClassification `?